rules/do-you-gather-more-personal-information-progressively/rule.md

```diff
@@ -3,22 +3,32 @@ type: rule
 archivedreason: 
 title: Do you gather more personal information progressively?
 guid: f9928c70-a007-4d02-972d-46fdde6db4db
-uri: do-you-gather-more-personal-information-progressively
+uri: gather-personal-information-progressively
 created: 2015-02-13T02:50:56.0000000Z
-authors: []
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects: []
 
 ---
 
-Most people don't like to disclose personal information and in some cases will give    you dud data. This is more often the case when people are required to give a lot    of information at one time. What we have found at SSW is that you will get better    quality personal information at intervals and not all at once.
+Most people are reluctant to disclose personal information and may provide false data, especially when asked to disclose a lot of information at once. It has been observed at SSW that collecting personal information in intervals, rather than in a single session, leads to more accurate responses.
```

Comment:
Hey @bettybondoc 

We have 2 times the word "disclose" very next to each other

- [ ] 1. Can you find a synonym and replace in either of the occurences?
- [ ] 2. We don't have data to confirm the statement below... so I would change from

> It has been observed at SSW that collecting personal information in intervals, rather than in a single session, leads to more accurate responses.

to

> Collecting personal information in intervals, rather than in a single session, may lead to more accurate and thoughtful answers.

Reply:
Thanks, @tiagov8!

> 1. Can you find a synonym and replace in either of the occurences?

Done - Changed From 

> Most people are reluctant to disclose personal information and may provide false data, especially when asked to disclose a lot of information at once.

To 

> Most people are reluctant to **_share_** personal information and may provide false data, especially when asked to disclose a lot of information at once.

> 2. We don't have data to confirm the statement below... so I would change from

Done